modules/images-configuration-parameters.adoc

```diff
@@ -33,11 +33,11 @@ The namespace for this config map is `openshift-config`. The format of the confi
 |Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and
 pods. For instance, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry.
 
-`insecureRegistries`: Registries which do not have a valid TLS certificate or only support HTTP connections. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+`insecureRegistries`: Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*``) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
```

Comment:
Did you intend to have the extra ` character?

` asterisk (*`) wildcard character `

Reply:
Most definitely not! Fixed. Thanks!